unit_test/test.cpp

```diff
@@ -2,6 +2,7 @@
 
 #include <Cabana_Core.hpp>
 #include <Cabana_Grid.hpp>
+#include <CommandLine.hpp>
```

Comment:
I think it's worth splitting these tests into a separate file (and renaming the existing test file) - maybe `CabanaPF_Test_PFHub` and `CabanaPF_Test_CommandLine`?